<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistPageForm</class>
 <widget class="QWidget" name="PlaylistPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playlist</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <widget class="QWidget" name="scrollContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>798</width>
        <height>600</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="contentLayout">
       <property name="margin">
        <number>16</number>
       </property>
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QWidget" name="infoWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="infoLayout">
          <property name="margin">
           <number>16</number>
          </property>
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="coverLabel">
            <property name="minimumSize">
             <size>
              <width>160</width>
              <height>160</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>160</height>
             </size>
            </property>
            <property name="styleSheet">
             <string>QLabel {
    border: 2px solid #ddd;
    border-radius: 8px;
    background-color: #f5f5f5;
}</string>
            </property>
            <property name="text">
             <string>Cover</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="playlistInfoWidget">
            <layout class="QVBoxLayout" name="playlistInfoLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="playlistNameLabel">
               <property name="font">
                <font>
                 <pointsize>18</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Playlist Name</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="creatorLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string>QLabel { color: #666; }</string>
               </property>
               <property name="text">
                <string>Created by: Unknown</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="songCountLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string>QLabel { color: #666; }</string>
               </property>
               <property name="text">
                <string>0 songs</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalDurationLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string>QLabel { color: #666; }</string>
               </property>
               <property name="text">
                <string>Total: 00:00:00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="descriptionLabel">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string>QLabel { color: #888; }</string>
               </property>
               <property name="text">
                <string>No description available.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="infoVerticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="songListView">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Uploader/Artist</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Platform</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Duration</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>